tcms/rpc/tests/test_testexecution.py

```diff
@@ -334,6 +334,34 @@ def test_update_with_non_existing_assignee_username(self):
                 "assignee": 'nonExistentUsername'
             })
 
+    def test_update_with_numeric_case_text_version(self):
+        self._update_case_text_version()
+        latest_case_text_version = self.case_run_1.case.history.latest().history_id
+
+        execution = self.rpc_client.TestExecution.update(self.case_run_1.pk, {
+            "case_text_version": str(latest_case_text_version)
+        })
+        self.assertEqual(execution["case_text_version"], latest_case_text_version)
+
+    def test_update_with_latest_case_text_version(self):
+        self._update_case_text_version()
+        latest_case_text_version = self.case_run_1.case.history.latest().history_id
+
+        execution = self.rpc_client.TestExecution.update(self.case_run_1.pk, {
+            "case_text_version": 'latest'
+        })
+        self.assertEqual(execution["case_text_version"], latest_case_text_version)
+
+    def _update_case_text_version(self):
+        initial_case_text_version = self.case_run_1.case_text_version
+        self.rpc_client.TestCase.update(self.case_run_1.case.pk, {
+            'summary': 'This was updated',
+            'text': 'new TC text',
+        })
```

Comment:
Here you don't really need to use the API to update the underlying TC object. You can call self.case_run_1.case.summary = 'This was updated' and then save it! We know that this works and if need be it is tested elsewhere so you don't really need to assert either. Not here at least.